### PR TITLE
Params to Funnel

### PR DESCRIPTION
**Pull request by ksalmon**

### CODE REVIEW

File: src/Widget.tsx [hunk 1]
```diff
@@ -32,85 +32,80 @@ const Wrapper = styled.div`
...
+      email,
+      phone,
+    });
 
-      setSubmitting(false)
-      return false;
-    }
-  }
+    const formQuery = new URLSearchParams(data);
+    const redirect = `https://find.solace.health/?${formQuery}`;
```
Comment (thomasmarren) [inline src/Widget.tsx:77]: Can we remove this slash?

```suggestion
    const redirect = `https://find.solace.health?${formQuery}`;
```

File: src/Widget.tsx [hunk 2]
```diff
@@ -32,85 +32,80 @@ const Wrapper = styled.div`
...
+    console.log(redirect);
+    window.location.assign(redirect);
```
Comment (thomasmarren) [inline src/Widget.tsx:79]: Did you want to keep this assign or `location.href = redirect`? Not sure which is better

**Review by thomasmarren — APPROVED** [review]
Tested with my [funnel changes](https://github.com/Solace-Health/solace-health-frontend/pull/65) and works like a charm 🤞

**Comment by ksalmon** [comment]
> Did we want to add the analytics.track to the funnel for FUNNEL_ENTRY as well?

@thomasmarren I would say yes, should we remove it from here?

**Comment by thomasmarren** [comment]
> > Did we want to add the analytics.track to the funnel for FUNNEL_ENTRY as well?
> 
> @thomasmarren I would say yes, should we remove it from here?

It looks like you did which is why I asked.

I do also think it's a good idea so we have no requests in the handleSubmit. I'll add it to me PR

File: src/Widget.tsx (hunk 2 above)
Comment (ksalmon) [reply src/Widget.tsx:79]: Assign and Href changes both add to the History layer. So I'm fine with either. Href might be faster from what I have heard

**Pull request merged**